Docs/quickstart.md

```diff
@@ -1,376 +0,0 @@
-# Quick Start
```

Comment:
quickstart.md is still here but now empty?

Reply:
removed